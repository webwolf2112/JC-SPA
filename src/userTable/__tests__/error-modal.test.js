import React from 'react';
import { shallow } from 'enzyme';
import ErrorModal from '../error-modal';

describe( 'Error Modal ', () => {
    const defaultProps = {
        errorMessage: 'test error message',
        dismissError: jest.fn(),
    };

    it('Should render correct', () => {
        const wrapper = shallow(
          <ErrorModal {...defaultProps} />,
        );
      
        expect(wrapper).toMatchSnapshot();
      });
});

