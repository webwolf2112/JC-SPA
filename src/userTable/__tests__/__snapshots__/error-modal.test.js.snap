// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Modal  Should render correct 1`] = `
<div
  className="modal"
>
   
  test error message
   
</div>
`;
